crates/cli/src/main.rs

```diff
@@ -57,23 +101,44 @@ fn main() -> Result<()> {
         }
     }
 
-    let interface_path = PathBuf::from(&opts.interface_file);
-    create_shims(&interface_path, &export_shim_path)?;
+    // Create our shim files given our parsed TS module object
+    // We have a shim file for exports and one optional one for imports
+    generate_wasm_shims(
+        plugin_interface.exports,
+        &export_shim_path,
+        plugin_interface.imports,
+        &import_shim_path,
+    )?;
 
+    // Merge the export shim with the core module
     let mut command = Command::new("wasm-merge")
         .arg(&core_path)
         .arg("coremod")
         .arg(&export_shim_path)
         .arg("codemod")
         .arg("-o")
-        .arg(&opts.output)
+        .arg(&linked_shim_path)
         .spawn()?;
     let status = command.wait()?;
```

Comment:
TODO: need a better error message for when wasm-merge can't be found on the machine